frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -28,6 +28,7 @@ declare global {
       selectProject(project: string): void;
       createNAD(namespace: string): void;
       waitForLoginPrompt(vmName: string, namespace: string): void;
+      visitNAD(): void;
```

Comment:
I think `visitNADPage` would be a better/clearer name, but I'm not going to hold up this PR for that.